src/epochs.lua

```diff
@@ -17,6 +17,12 @@ local epochs = {}
 --- @field startHeight number The start height of the epoch
 --- @field distributionTimestamp number The distribution timestamp of the epoch
 --- @field observations Observations The observations of the epoch
+--- @field arnsStats ArNSStats The ArNS stats for the epoch
+
+--- @class ArNSStats # The ArNS stats
+--- @field totalActiveNames number The total active ArNS names
+--- @field totalReservedNames number The total reserved ArNS names
```

Comment:
we could add `totalGracePeriodNames` as well since we're iterating through all the records for active names

Reply:
added!